### DISCUSSION TITLE
Cursor offset when resizing Region on zoomed in element

### DISCUSSION THREAD

**Discussion by benwager** [katspaugh/wavesurfer.js#3441] [Q&A] [answered]
I am embedding a Wavesurfer component within another element in my application. The user can zoom in and out of the entire application. So I am not using the Zoom plugin of wavesurfer. 

When the page has been zoomed, this creates an offset when dragging the Region handles to resize a region. 

Is there any way, without having to modify the Regions plugin, to account for the current zoom level?

https://github.com/katspaugh/wavesurfer.js/assets/5874704/26381f18-b3e6-4b87-9be8-d4342cd01e1f

**Comment by katspaugh** [comment, marked as answer]
Fixed in #3442 👍

**Reply by benwager** [reply]
Damn, that was quick. Just updated package and confirm it is now working as expected. Thanks!!
